pylint/checkers/utils.py

```diff
@@ -1919,24 +1919,29 @@ def in_type_checking_block(node: nodes.NodeNG) -> bool:
     return False
 
 
-def is_typing_literal(node: nodes.NodeNG) -> bool:
-    """Check if a node refers to typing.Literal."""
+def is_typing_member(node: nodes.NodeNG, typing_members: str | tuple[str, ...]) -> bool:
```

Comment:
```suggestion
def is_typing_member(node: nodes.NodeNG, names_to_check: tuple[str, ...]) -> bool:
```

I now understand the function name, let's just makes the argument a little clearer.

I would also suggest to do the tuple conversion outside of this helper: we're changing calls anyway, might as well make this handle `tuples` only.